File: python/recorridoMatriz.py
matriz = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
]

print("Recorrido por filas:")

for i in range(len(matriz)):     
    for j in range(len(matriz[i])): 
        print(matriz[i][j], end=" ")

print()
print("Recorrido por columnas:")

for i in range(len(matriz)):     
    for j in range(len(matriz[i])): 
        print(matriz[j][i], end=" ")

File: c++/recorridoMatriz.cpp
#include <iostream>
using namespace std;

int main() {

    int matriz[3][3] = {
        {1, 2, 3},
        {4, 5, 6},
        {7, 8, 9}
    };
    
    cout << "Recorrido por filas:" << endl;

    for(int i = 0; i < 3; i++) {       
        for(int j = 0; j < 3; j++) {     
            cout << matriz[i][j] << " ";
        }
    }

    cout <<endl << "Recorrido por columnas:" << endl;

    for(int i = 0; i < 3; i++) {       
        for(int j = 0; j < 3; j++) {     
            cout << matriz[j][i] << " ";
        }
    }

    return 0;
}